README.md

```diff
@@ -1 +1,217 @@
-# Final-group-capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[final-group-capstone \] ](#-final-group-capstone--)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [Kanban Board ](#kanban-board-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [final-group-capstone ] <a name="about-project"></a>
+
+**[ final-group-capstone ]** is an application which allows users to to book an apponitment with a doctor for the    stipulated amount of time.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Authenicate user]**
+- **[Fetch bookings]**
+- **[Post bookings]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Have ruby installed
+- Have node installed
+- Have rails installed
+
+### Setup
+
+You can find the React Frontend client at [React-client](https://github.com/Ibtisam34/Final-group-capstone-frontend.git)
+
+Clone this repository to your desired folder:
+
+To run and view this application, you will first have to clone a local version of this repository using:
+
+git clone https://github.com/Ibtisam34/Final-group-capstone
+
+### Install
+
+Install this project with:
+ `bundle install`
+
+### Usage
+
+- To run the project follow the instructions below:
+
+- You can find the React Frontend client at [React-client](https://github.com/Ibtisam34/Final-group-capstone-frontend)
+
+- Configure Environment file in your project by running the following command:
+
+`bundle exec figaro install`
```

Comment:
- [ ] This command still won't work without the proper gem in the `gemfile`.  Kindly add the gem to said file, or instruct your users to add it and install it before using the command. 👍